connect-example/src/main/java/com/stripe/android/connect/example/Theme.kt

```diff
@@ -70,50 +58,3 @@ fun MainContent(
         }
     }
 }
-
-@OptIn(ExperimentalMaterialApi::class)
-@Composable
-fun BasicComponentExampleContent(
```

Comment:
👀 Moved to connect-example/src/main/java/com/stripe/android/connect/example/ui/common/BasicComponentExample.kt